packages/react-native-drawer-layout/src/views/GestureHandler.tsx

```diff
@@ -1,19 +1,11 @@
 import * as React from 'react';
 import { View } from 'react-native';
-import type {
-  PanGestureHandlerProperties,
-  TapGestureHandlerProperties,
-} from 'react-native-gesture-handler';
 
 const Dummy: any = ({ children }: { children: React.ReactNode }) => (
   <>{children}</>
 );
 
-export const PanGestureHandler =
-  Dummy as React.ComponentType<PanGestureHandlerProperties>;
-
-export const TapGestureHandler =
-  Dummy as React.ComponentType<TapGestureHandlerProperties>;
+export const GestureDetector = Dummy as any; // awkward, but this is not very important
```

Comment:
This file will be used for the actual types so important to not use `any`.

```suggestion
export const GestureDetector = Dummy as typeof import('react-native-gesture-handler').GestureDetector;
```